calico_node/startup/startup.go

```diff
@@ -542,20 +547,35 @@ func ipv6Supported() bool {
 
 // createIPPool creates an IP pool using the specified CIDR.  This
 // method is a no-op if the pool already exists.
-func createIPPool(client *client.Client, cidr *net.IPNet, ipip bool) {
+func createIPPool(client *client.Client, cidr *net.IPNet, ipipModeName string) {
 	version := cidr.Version()
+	ipipMode := ipip.Mode(ipipModeName)
+
+	// off is not an actual valid value so switch it to an empty string
+	if "off" == ipipModeName {
```

Comment:
NP, not sure why I did it that way, probably a holdover from it doing that way in C++ so the compiler would complain if it was an assignment to a constant instead of the equality operator.